packages/terra-date-picker/src/react-datepicker/index.jsx

```diff
@@ -499,10 +506,8 @@ class DatePicker extends React.Component {
 
   handleCalendarClickOutside = (event) => {
     if (!this.props.inline) {
-      this.setOpen(false)
+      this.setOpen(false, event)
     }
-    this.props.onClickOutside(event)
-    if (this.props.withPortal) { event.preventDefault() }
```

Comment:
We may need to keep this depending on this [decision](https://github.com/cerner/terra-framework/pull/1224/files#r476671982).

Reply:
Updated here https://github.com/cerner/terra-framework/pull/1224/commits/9d8be2a715656afc67046abb501756e8ef610399